cfme/common/vm.py

```diff
@@ -390,11 +391,12 @@ def load_details(self, refresh=False):
             sel.click(self.find_quadicon())
         else:
             if refresh:
-                # bz1392139 for 5.7, should be fixed in 5.7.1 - dajo
-                if version.current_version() < "5.7":
-                    toolbar.refresh()
-                else:
+                # bz1389299 for 5.7, should be fixed in 5.7.1 - dajo
+                reload_bug = BZ(1329299)
+                if reload_bug.bugzilla.get_bug(1329299).is_opened:
```

Comment:
will try this way of blocking. but it's actually not perspective in future. this just check only on hardcoded BZ, but DEVs have already closed 2 on this issue